tools/Az/Az.psd1

```diff
@@ -155,120 +155,123 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '2.0.0 - May 2019
-General
-* This iis a breaking change release.  A detailed migration guide is available at: https://aka.ms/azps-migration-latest
-Az.Accounts
-* Update Authentication Library to fix ADFS issues with username/password auth
-
-Az.CognitiveServices
-* Only display Bing disclaimer for Bing Search Services.
-* Improve error when create account failed.
+        ReleaseNotes = '2.1.0 - May 2019
```

Comment:
This raises a red flag about the size of the 'release notes' field.  I think it is OK as is, but we won't know for sure until we push to the TestGallery.  We should be prepared to crack open the package and change the nuspec file if that occurs, or shorten the ApiManagement entry now.